stacks-signer/src/signerdb.rs

```diff
@@ -308,6 +308,10 @@ static CREATE_INDEXES_3: &str = r#"
 CREATE INDEX IF NOT EXISTS block_rejection_signer_addrs_on_block_signature_hash ON block_rejection_signer_addrs(signer_signature_hash);
 "#;
 
+static CREATE_INDEXES_4: &str = r#"
+CREATE INDEX IF NOT EXISTS blocks_state ON blocks ((json_extract(block_info, '$.state')));
```

Comment:
Bit of a side note, but I think it might be best to make this query on both the state and the consensus hash since we never really ever query blocks without both.

Reply:
Should we do (maybe this is overindexed...): 
```
"static CREATE_INDEXES_4: &str = r#"
CREATE INDEX IF NOT EXISTS blocks_state ON blocks (consensus_hash, json_extract(block_info, '$.state'),  stacks_height DESC);
"#;
```